src/OrchardCore.Modules/OrchardCore.Media/Assets/js/MediaApp/fileupload.js

```diff
@@ -12,6 +12,21 @@ $(document).on('mediaApp:ready', function () {
                 { name: '__RequestVerificationToken', value: antiForgeryToken },
             ]
         },
+        change: function (e, data) {
```

Comment:
It seems that the 'change' event is not triggered when dropping files.
Maybe we could use the 'add' event. 
According to the docs it is triggered by drag and drop:
[blueimp docs for add event](https://github.com/blueimp/jQuery-File-Upload/wiki/Options#add)